src/pypdfium2/_helpers/pageobjects.py

```diff
@@ -278,7 +306,10 @@ def get_data(self, decode_simple=False):
         Returns:
             ctypes.Array: The data of the image stream (as :class:`~ctypes.c_ubyte` array).
         """
-        func = pdfium_c.FPDFImageObj_GetImageDataDecoded if decode_simple else pdfium_c.FPDFImageObj_GetImageDataRaw
+        func = {
+            False: pdfium_c.FPDFImageObj_GetImageDataRaw,
+            True: pdfium_c.FPDFImageObj_GetImageDataDecoded,
+        }[decode_simple]
```

Comment:
TODO: want to change these `{False: ..., True: ...}[var]` dicts back to `... if var else ...` after all. so any object whose `bool()` is True can be used.
Also it seems like overkill to create a dict just for True/False.